UnityProject/Assets/Light2D/Editor/LightingSystemEditor.cs

```diff
@@ -67,12 +63,14 @@ public override void OnInspectorGUI()
             serializedObject.Update();
 
             if (Application.isPlaying)
+            {
                 GUI.enabled = false;
+            }
 
             var lightingSystem = (LightingSystem) target;
             var cam = lightingSystem.GetComponent<Camera>();
-            bool isMobileTarget = EditorUserBuildSettings.activeBuildTarget == BuildTarget.iOS ||
-                                  EditorUserBuildSettings.activeBuildTarget == BuildTarget.Android;
+            var isMobileTarget = EditorUserBuildSettings.activeBuildTarget == BuildTarget.iOS ||
```

Comment:
Changing all the bools/int/vector3/whatever into vars doesn't help I don't think... It decreases readability by making it harder to check what type a variable is.

Reply:
Thank you, will change it! :)